inference/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM quay.io/azavea/raster-vision:pytorch-v0.12.0
```

Comment:
What's the story for getting the upstream raster-vision pytorch container updated to newer pystac versions? Is there an open issue to document this?

Reply:
https://github.com/azavea/raster-vision/issues/968